lib/tandem/tandemSimulator.js

```diff
@@ -79,6 +79,22 @@ exports.make = function(config){
 
   return {
     alarm: function(event) {
+      if (currStatus !== null && currStatus.status === 'suspended') {
+        var type = event.alarmType;
+        if(type === 'occlusion' || type === 'auto_off' || type === 'no_insulin' || type === 'no_power' ||
```

Comment:
The only one I could check this with in my data was occlusion, but I saw suspended.reason = 'automatic' as expected :)